docs/low-code-jobs/airflow/setup/mwaa.md

```diff
@@ -76,8 +76,7 @@ For simplicity, you can switch to **(3) JSON** and use below JSON policy and rep
         "s3:PutObject",
         "s3:GetObject",
         "airflow:CreateCliToken",
```

Comment:
@saisrikarbabu should the CreateCliToken also be moved above?
I am not able to test this policy directly. Can you pls confirm it's tested?
Thanks in advance!

Reply:
no only getenvironment need to be moved above.